assets/src/stories-editor/blocks/amp-story-page/index.js

```diff
@@ -51,6 +51,10 @@ const schema = {
 	overlayOpacity: {
 		default: 50,
 	},
+	isExcessiveVideoSize: {
```

Comment:
I don't think this should be an attribute that's saved with the block.

Whether a video is excessive or not can be determined at runtime by checking the media object.

Reply:
Ah, OK. So this notice doesn't need to display on reloading the page, only right after selecting the video?

Reply:
Oh, you're mentioning that whether the video size is excessive could be calculated at runtime, without saving this attribute.

Reply:
Yes exactly.

Reply:
Hm, I worked on doing this calculation inside the [render() method](https://github.com/ampproject/amp-wp/blob/b342ff33aec3da9a62ce06d33535867461a238d5/assets/src/stories-editor/blocks/amp-story-page/edit.js#L200) of `amp-story-page/edit.js`, so that `isExcessiveVideoSize` doesn't have to be stored in an attribute. 

But I can't get the `media` object of the video in that `render()` method on page load.

This:
```js
select( 'core' ).getMedia( mediaId )
```
...returns `undefined`

I'll look at this a little more.

Reply:
That select needs to be run in a withSelect wrapper. It will return undefined at first, but as soon as the data arrives it will be there and the component will be re-rendered.

Reply:
Ah, good idea.

Reply:
Thanks, https://github.com/ampproject/amp-wp/pull/2509/commits/537fb08d7d5eb8d4465c8bbc6416acd840849cf1 uses your idea. It works locally 😄